packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -57,11 +63,28 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       });
 
       this.webCallingService = new WebCallingService(this.$webex, this.$config.callingClientConfig);
+      this.taskManager = TaskManager.getTaskManager(
+        this.services.contact,
+        this.webCallingService,
+        this.services.webSocketManager
+      );
 
       LoggerProxy.initialize(this.$webex.logger);
     });
   }
 
+  private handleIncomingTask = (task: ITask) => {
+    // @ts-ignore
```

Comment:
Let's create a jira or mention that this needs to be handled in a future PR. We should avoid ts-ignore as much as possible and see if there's another way.